samples/accountLinking/skill/states.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+exports.register = function register(skill) {
```

Comment:
is this part of the PR?

Reply:
This looked like it was merged directly to master instead of DEV, so now that I try to sync branches, I had to include it as part of the conflicts